packages/jest-environment-jsdom/src/index.js

```diff
@@ -44,7 +44,18 @@ class JSDOMEnvironment {
     }
 
     this.moduleMocker = new mock.ModuleMocker(global);
-    this.fakeTimers = new FakeTimers(global, this.moduleMocker, config);
+
+    const timerConfig = {
+      idToRef: (id: number) => id,
+      refToId: (ref: number) => ref,
+    };
+
+    this.fakeTimers = new FakeTimers(
```

Comment:
Can we make the constructor take an object instead of multiple arguments?

Reply:
Good point! Updated.